lib/core/src/client/preview/start.js

```diff
@@ -219,5 +219,62 @@ export default function start(render, { decorateStory } = {}) {
     window.__STORYBOOK_ADDONS_CHANNEL__ = channel; // may not be defined
   }
 
-  return { context, clientApi, configApi, forceReRender };
+  let previousExports = {};
+  const loadStories = (req, framework) => () => {
+    req.keys().forEach(filename => {
+      const fileExports = req(filename);
+
+      // An old-style story file
+      if (!(fileExports.default && fileExports.default.title)) {
```

Comment:
Should we throw an error or something if there is a default export without `title`?